docs/03-dev-howtos/13-Update-configuration-in-s3.md

```diff
@@ -12,35 +12,35 @@ To add or update a configuration item you need to:
 aws s3 ls --profile=frontend s3://aws-frontend-store/config/
 ```
 
-look for the most recent version ( the `v14` part ):
+look for the most recent version ( the `v17` part ):
```

Comment:
I'm not sure continuing to bump this every time is particularly valuable? Clearly, I never remember anyway! 🙈